widgetastic_manageiq/__init__.py

```diff
@@ -2059,7 +2103,7 @@ class Search(View):
     """ Represents search_text control
     # TODO Add advanced search
     """
-    search_text = Input(name="search_text")
+    search_text = Input(id="search_text")
```

Comment:
Q: did you double check that it is true for 5.7?

Reply:
It's the the same, true.